### PR TITLE
Add New Roadmap

### PR DESCRIPTION
**Pull request by amanifarooque**
Preview Link: https://cg-a72b65a7-73f0-4859-a1d7-dc953cf6ade8.sites.pages.cloud.gov/preview/gsa/search-gov-website/fy23-q2-roadmap/about/updates/roadmap/ 

## PR Checklist
- [x] Test Cloud.gov Pages preview link for your branch for functionality - make sure all links and buttons work, javascript works as intended, and updates display as expected
- [N/A] Conduct a HemingwayApp review of the content for plain language. Request plain language review in #helpwanted if needed before merging this PR
- [N/A] Conduct a Lighthouse Audit via Chrome Developer Tools. Identify any issues that must be addressed before merging, and fix
- [x] Ensure no non-public information is published in the commit messages

### CODE REVIEW

File: about/updates/roadmap/FY23Q1-roadmap.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+### Now
+
+* **Faceted Search &mdash; Results API Support**
+  * We will work on supporting the faceted search functionality in our Results API.
+* **Javascript Indexing &mdash; Production Release**
+  * We will be getting our Javascript indexing feature tested and ready for production.
+* **Search Engine Results Page (SERP) Redesign &mdash; Investigations**
+  * We will begin scoping the work required to upgrade our frontend technology in support of this redesign.
+* **Version Upgrades**
+  * We will be upgrading our ElasticSearch, Ruby, Rails, and Redis versions to keep them up-to-date and compliant.
```
Comment (dawnpm) [inline about/updates/roadmap/FY23Q1-roadmap.md:34]: oops number disagreement "upgrading our Ruby version to keep them"

File: about/updates/roadmap/index.md [hunk 2]
```diff
@@ -22,30 +22,23 @@ We know it's helpful for our partners at federal agencies to know what's coming
...
-  * We will begin scoping the work required to upgrade our frontend technology in support of this redesign.
+* **Search Engine Results Page (SERP) Redesign &mdash; Frontend Development**
+  * We will continue frontend development to build and test the new results page designs.
 * **Version Upgrades**
-  * We will be upgrading our ElasticSearch, Ruby, Rails, and Redis versions to keep them up-to-date and compliant. 
-* **RSS Feed Indexing**
-  * In addition to sitemap indexing, we will add support for automated indexing of RSS feeds into our primary web index.
-* **Additional Language Support**
-  * We will be adding support to our in-house index for Bengali, Hindi, Tagalog, Khmer, Vietnamese, Navajo, and Yu'pik search experiences.
+  * We will be upgrading our Ruby version to keep them up-to-date and compliant.
```
Comment (dawnpm) [inline about/updates/roadmap/index.md:34]: number disagreement again

**Review by dawnpm — CHANGES_REQUESTED** [review]
couple of ds_store changes snuck in. also, one grammar oops from last time now exists in both files

File: about/updates/roadmap/FY23Q1-roadmap.md (hunk 1 above)
Comment (amanifarooque) [reply about/updates/roadmap/FY23Q1-roadmap.md:34]: > We will be upgrading our ElasticSearch, Ruby, Rails, and Redis versions to keep them up-to-date and compliant. 

Versions are plural, shouldn't it be them?

File: about/updates/roadmap/index.md (hunk 2 above)
Comment (amanifarooque) [reply about/updates/roadmap/index.md]: Fixed

**Comment by amanifarooque** [comment]
@dawnpm I'm deleting DS store files that snuck in previously, so the changes in this PR should remain to remove them. I only see one number mismatch; can you clarify why the other needs to change?

**Review by dawnpm — APPROVED** [review]

**Pull request merged**